advocacy_docs/edb-postgres-ai/migration-etl/index.mdx

```diff
@@ -9,34 +9,56 @@ navigation:
-The AI copilot is designed to help users with the following tasks:
+- **Purpose:** Migration of database data
 
-- **General migration assistance**: The AI copilot can help users with general migration questions. 
-  For example, users can request information about available tools, and source and target database compatibility.
-- **Migration planning**: The AI copilot can help users plan their migration, and obtain an overview of the end-to-end migration paths.
-- **Migration assessment**: The AI copilot can help users assess their migration readiness. 
-  For example, if there are compatibility issues between source and target databases, the AI Copilot can suggest compatible query alternatives. 
+- **Applicable migration scenarios:** From Oracle, PostgreSQL or EDB Postgres Advanced Server to [EDB Hosted Cloud Service](/edb-postgres-ai/cloud-service/getting_started/planning/choosing_your_deployment/edb_hosted_cloud_service/) databases with PostgreSQL, EDB Postgres Advanced Server or EDB Postgres Extended Server
 
-The AI copilot is designed to be user-friendly and easy to use. Users can interact with the AI copilot using natural language and improve the quality of answers with [good prompting](/migration_portal/latest/03_mp_using_portal/mp_ai_copilot/ai_good_prompts/). The AI copilot is also designed to be context-aware, so it can provide users with relevant information based on the context of the conversation.
+## EDB Postgres AI Migration Portal
 
-## The EDB Postgres AI Replication Server
+[EDB Postgres AI Migration Portal](/migration_portal/latest/) is a web-based tool that helps you assess and analyze Oracle database schemas and convert them to EDB Postgres Advanced Server.
 
-The [EDB Postgres Replication Server](/eprs/latest/) is a tool that helps you keep your Postgres database up-to-date with changes from other databases.
+EDB Postgres® AI Migration Portal offers an [AI copilot](/migration_portal/latest/03_mp_using_portal/mp_ai_copilot/) to assist users who are migrating their databases to EDB Postgres Advanced Server.
 
-## The EDB Postgres AI Migration Toolkit
+- **Purpose:** Conversion and import of schemas
 
-The [EDB Postgres Migration Toolkit](/migration_toolkit/latest/) is a set of tools that help you migrate your data from other databases to Postgres.
+- **Applicable migration scenarios:** From Oracle to EDB Postgres Advanced Server
+
+## EDB Postgres AI Migration Toolkit
+
+[EDB Postgres Migration Toolkit](/migration_toolkit/latest/) is a set of tools that help you migrate database objects and data at a granular level from non-EDB databases to EDB Postgres.
+
+- **Purpose:** Migration of database objects and data, includes options for more granular control of the process.
+
+- **Applicable migration scenarios:** From Oracle, MySQL, Microsoft SQL Server, Sybase Adaptive Server Enterprise or Postgresql to EDB Postgres Advanced Server or PostgreSQL.
+
+## EDB Oracle Migration Handbook 
```

Comment:
I would move the Migration Handbook to be the last section on this page since all of the other sections describe software components we offer.